exonum-java-proofs/src/test/java/com/exonum/binding/storage/proofs/map/flat/CheckedFlatMapProofTest.java

```diff
@@ -30,11 +28,11 @@ public void mapProofShouldBeValid() {
 
     DbKey valueKey = DbKey.newLeafKey(secondKey);
     MapProofEntryLeaf leaf = createLeafMapEntry(valueKey.getKeySlice(), VALUE);
-    List<MapProofEntry> entries =
-        Arrays.asList(
-            createBranchMapEntry(firstKey),
-            leaf,
-            createBranchMapEntry(thirdKey));
+    MapProofEntry[] entries = {
```

Comment:
I'd keep using lists here — (1) they are more convenient; (2) tests might not need any sorting.